src/main/xml-parsers.js

```diff
@@ -459,4 +459,9 @@ export function parseListObjectsV2WithMetadata(xml) {
 export function parseBucketVersioningConfig(xml){
   var xmlObj = parseXml(xml)
   return xmlObj.VersioningConfiguration
+}
+
+export function parseTagging(xml){
+  const xmlObj = parseXml(xml)
+  return xmlObj
```

Comment:
`xmlObj` looks like this; `xmlObj = { Tagging: { TagSet: { Tag: [Array] } } }`

Do we really return the whole object with all these layers?

Why not return a cleaner version of it, `return xmlObj.Tagging.TagSet.Tag`?

The cleaner version would be much more similar to what is displayed in `mc tag list <alias/bucket[/path-to-object]>` command output.

Reply:
I will update it to return (`xmlObj.Tagging.TagSet.Tag` ) an array of objects like the below: 
```
[
  { Key: 'tag2', Value: 'tag value 2' },
  { Key: 'tag3', Value: 'tag value 3' },
  { Key: 'tag4', Value: 'tag value 4' }
]
```